lib/node_modules/@stdlib/blas/ext/base/scusumkbn/manifest.json

```diff
@@ -1,40 +1,50 @@
 {
-	"options": {},
-	"fields": [
-		{
-			"field": "src",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "include",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "libraries",
-			"resolve": false,
-			"relative": false
-		},
-		{
-			"field": "libpath",
-			"resolve": true,
-			"relative": false
-		}
-	],
-	"confs": [
-		{
-			"src": [
-				"./src/scusumkbn.c"
-			],
-			"include": [
-				"./include"
-			],
-			"libraries": [
-				"-lm"
-			],
-			"libpath": [],
-			"dependencies": []
-		}
-	]
+  "options": {
+    "task":"build"
+  },
+  "fields": [
+   {
+     "field": "src",
+     "resolve": true,
+     "relative": true	
+   },
+   {
+     "field": "include",
+     "resolve": true,
+     "relative": true	
+   },
+   {
+     "field": "libraries",
+     "resolve": false,
+     "relative": false	
+   },
+   {
+     "field": "libpath",
+     "resolve": true,
+     "relative": false	
+   }
+  ],
+  "confs": [
+   {
+     "task": "build",
+     "src": [
+       "./src/scusumkbn.c"
+     ],
+     "include": [
+      "./include"
+     ],
+     "libraries": [
+      "-lm"
+     ],
+     "libpath": [],
+     "dependencies": [
+        "@stdlib/napi/export",
+        "@stdlib/napi/argv",
+        "@stdlib/napi/argv-float",
+        "@stdlib/napi/argv-int64",
+        "@stdlib/napi/argv-strided-float32array"
+     ]
+   }
+  ]
 }
```

Comment:
You still need the `benchmark` and `examples` configurations. Those need to be re-added. They just don't have dependencies.